packages/babel-helpers/src/helpers/regeneratorRuntime.js

```diff
@@ -340,7 +340,7 @@ export default function _regeneratorRuntime() {
     var method = delegate.iterator[methodName];
     if (method === undefined) {
       // A .throw or .return when the delegate iterator has no .throw
-      // method, or a missing .next mehtod, always terminate the
+      // method, or a missing .next method, always terminate the
```

Comment:
This file is auto-generated from upstream repo, please open a similar PR to https://github.com/facebook/regenerator.

Reply:
https://github.com/facebook/regenerator/pull/679